runtime/onert/core/include/exec/Executors.h

```diff
@@ -19,6 +19,22 @@
 
 #include "IExecutor.h"
 #include "ir/NNPkg.h"
+#include "util/Index.h"
+
+namespace std
+{
+
+template <> struct hash<std::pair<::onert::ir::ModelIndex, ::onert::ir::SubgraphIndex>>
+{
+  size_t
+  operator()(const std::pair<::onert::ir::ModelIndex, ::onert::ir::SubgraphIndex> &pair) const
+    noexcept
+  {
+    return hash<uint64_t>()((uint64_t(pair.first.value()) << 32) | uint64_t(pair.second.value()));
```

Comment:
(beyond this pr) As I mentioned in https://github.com/Samsung/ONE/pull/9716#discussion_r975996904, we may reduce bit width i32 to i16 for both `ModelIndex` and `SubgraphIndex`, which are unlikely to be beyond 2^<sup>15</sup>.

Reply:
Basically right. But we need to think about tflite and circle schema, and NNAPI spec. They are using 32bit for subgraph index. (we can reduce model index bit)